### DISCUSSION TITLE
use IsGranted attribute in PHP 8

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#42810] [Q&A] [answered]
Hi
I want to use `@IsGranted("ROLE_ADMIN")` in my controller like this:

`#[IsGranted('ROLE_ADMIN')]`

But not work. Is it just need annotation?

** I use Symfony 5.3.7

**Comment by stof** [comment, marked as answer]
This annotation comes from SensioFrameworkExtraBundle, not from Symfony. So it depends on the version of the bundle you use.
Support for attributes was added in 6.1: https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/CHANGELOG.md

**Reply by karaka200** [reply]
Thank you
